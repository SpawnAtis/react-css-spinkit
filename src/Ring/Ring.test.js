import React from 'react'
import {render} from 'react-testing-library'
import Ring from './Ring'

describe('Ring component', () => {
  it('Should render Ring without problems', () => {
    const {asFragment} = render(<Ring />)
    expect(asFragment()).toMatchSnapshot()
  })
})
