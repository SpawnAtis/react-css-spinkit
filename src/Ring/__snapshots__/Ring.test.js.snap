// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ring component Should render Ring without problems 1`] = `
<DocumentFragment>
  <style>
    [object Object]
  </style>
  <div
    class="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
